plugins/resources-generator/src/main/kotlin/dev/icerock/gradle/generator/jvm/JvmImagesGenerator.kt

```diff
@@ -0,0 +1,37 @@
+package dev.icerock.gradle.generator.jvm
+
+import com.squareup.kotlinpoet.CodeBlock
+import com.squareup.kotlinpoet.KModifier
+import dev.icerock.gradle.generator.ImagesGenerator
+import org.gradle.api.file.FileTree
+import java.io.File
+
+class JvmImagesGenerator(inputFileTree: FileTree) : ImagesGenerator(inputFileTree) {
+
+    override fun getClassModifiers(): Array<KModifier> = arrayOf(KModifier.ACTUAL)
+
+    override fun getPropertyModifiers(): Array<KModifier> = arrayOf(KModifier.ACTUAL)
+
+    // TODO replace .png with getting extension
+    override fun getPropertyInitializer(key: String) =
+        CodeBlock.of("ImageResource(imagePath = %S)", "$IMAGES_DIR/$key.png")
```

Comment:
I think the only option is modifying the `ImagesGenerator` to get the file extension also

Reply:
maybe we just should pass into this function not only key but also full value (filename in this case)

Reply:
Fixed with 4a087fb97228eb2cb5697fde3e3d1e09ee5fef4e